dds/tests/write_read_samples.rs

```diff
@@ -44,6 +44,73 @@ struct LargeData {
     value: Vec<u8>,
 }
 
+

```

Comment:
I guess this didn't really belong to this change :) It is not a problem to get it in so long as it works